/mnt/hwfile/huqinghao/miniconda3/bin/python
2024-05-17 06:50:47,695 - INFO - Note: detected 128 virtual cores but NumExpr set to maximum of 64, check "NUMEXPR_MAX_THREADS" environment variable.
2024-05-17 06:50:47,695 - INFO - Note: NumExpr detected 128 cores but "NUMEXPR_MAX_THREADS" not set, so enforcing safe limit of 8.
2024-05-17 06:50:47,695 - INFO - NumExpr defaulting to 8 threads.
2024-05-17 06:50:48,989	INFO scripts.py:1182 -- Did not find any active Ray processes.

==========
== CUDA ==
==========

CUDA Version 12.1.0

Container image Copyright (c) 2016-2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.

This container image and its contents are governed by the NVIDIA Deep Learning Container License.
By pulling and using the container, you accept the terms and conditions of this license:
https://developer.nvidia.com/ngc/nvidia-deep-learning-container-license

A copy of this license is made available in this container at /NGC-DL-CONTAINER-LICENSE for your convenience.

*************************
** DEPRECATION NOTICE! **
*************************
THIS IMAGE IS DEPRECATED and is scheduled for DELETION.
    https://gitlab.com/nvidia/container-images/cuda/blob/master/doc/support-policy.md

Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63c6d0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63d450>: Failed to establish a new connection: [Errno 111] Connection refused'))
INFO 05-17 06:51:03 api_server.py:191] vLLM API server version 0.3.3
INFO 05-17 06:51:03 api_server.py:192] args: Namespace(host=None, port=8000, uvicorn_log_level='info', allow_credentials=False, allowed_origins=['*'], allowed_methods=['*'], allowed_headers=['*'], api_key=None, served_model_name=None, lora_modules=[], delta_modules=[Delta(name='delta-1', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-1'), Delta(name='delta-2', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-2'), Delta(name='delta-3', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-3'), Delta(name='delta-4', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-4'), Delta(name='delta-5', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-5'), Delta(name='delta-6', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-6'), Delta(name='delta-7', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-7'), Delta(name='delta-8', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-8'), Delta(name='delta-9', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-9'), Delta(name='delta-10', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-10'), Delta(name='delta-11', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-11'), Delta(name='delta-12', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-12'), Delta(name='delta-13', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-13'), Delta(name='delta-14', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-14'), Delta(name='delta-15', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-15'), Delta(name='delta-16', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-16'), Delta(name='delta-17', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-17'), Delta(name='delta-18', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-18'), Delta(name='delta-19', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-19'), Delta(name='delta-20', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-20'), Delta(name='delta-21', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-21'), Delta(name='delta-22', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-22'), Delta(name='delta-23', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-23'), Delta(name='delta-24', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-24'), Delta(name='delta-25', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-25'), Delta(name='delta-26', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-26'), Delta(name='delta-27', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-27'), Delta(name='delta-28', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-28'), Delta(name='delta-29', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-29'), Delta(name='delta-30', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-30'), Delta(name='delta-31', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-31'), Delta(name='delta-32', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-32'), Delta(name='delta-33', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-33'), Delta(name='delta-34', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-34'), Delta(name='delta-35', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-35'), Delta(name='delta-36', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-36'), Delta(name='delta-37', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-37'), Delta(name='delta-38', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-38'), Delta(name='delta-39', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-39'), Delta(name='delta-40', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-40'), Delta(name='delta-41', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-41'), Delta(name='delta-42', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-42'), Delta(name='delta-43', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-43'), Delta(name='delta-44', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-44'), Delta(name='delta-45', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-45'), Delta(name='delta-46', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-46'), Delta(name='delta-47', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-47'), Delta(name='delta-48', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-48'), Delta(name='delta-49', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-49'), Delta(name='delta-50', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-50'), Delta(name='delta-51', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-51'), Delta(name='delta-52', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-52'), Delta(name='delta-53', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-53'), Delta(name='delta-54', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-54'), Delta(name='delta-55', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-55'), Delta(name='delta-56', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-56'), Delta(name='delta-57', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-57'), Delta(name='delta-58', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-58'), Delta(name='delta-59', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-59'), Delta(name='delta-60', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-60'), Delta(name='delta-61', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-61'), Delta(name='delta-62', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-62'), Delta(name='delta-63', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-63'), Delta(name='delta-64', local_path='/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-64')], swap_modules=[], chat_template=None, response_role='assistant', ssl_keyfile=None, ssl_certfile=None, ssl_ca_certs=None, ssl_cert_reqs=0, root_path=None, middleware=[], model='/vllm/.idea/full_models/Llama-2-13b-hf', tokenizer=None, revision=None, code_revision=None, tokenizer_revision=None, tokenizer_mode='auto', trust_remote_code=False, download_dir=None, load_format='auto', dtype='auto', kv_cache_dtype='auto', max_model_len=None, worker_use_ray=False, pipeline_parallel_size=1, tensor_parallel_size=4, max_parallel_loading_workers=None, ray_workers_use_nsight=False, block_size=16, enable_prefix_caching=False, seed=0, swap_space=4, gpu_memory_utilization=0.85, max_num_batched_tokens=None, max_num_seqs=256, max_logprobs=5, disable_log_stats=False, quantization=None, enforce_eager=True, max_context_len_to_capture=8192, disable_custom_all_reduce=False, tokenizer_pool_size=0, tokenizer_pool_type='ray', tokenizer_pool_extra_config=None, enable_lora=False, max_loras=1, max_lora_rank=64, lora_extra_vocab_size=256, lora_dtype='auto', max_cpu_loras=32, enable_delta=True, max_deltas=24, max_cpu_deltas=32, max_delta_bitwidth=4, device='auto', image_input_type=None, image_token_id=None, image_input_shape=None, image_feature_size=None, scheduler_delay_factor=0.0, enable_prefetch=True, scheduler_policy='fcfs', max_swap_slots=1, max_cpu_models=4, enable_swap=False, engine_use_ray=False, disable_log_requests=True, max_log_len=None)
2024-05-17 06:51:06,589	INFO worker.py:1749 -- Started a local Ray instance.
INFO 05-17 06:51:08 llm_engine.py:90] Initializing an LLM engine (v0.3.3) with config: model='/vllm/.idea/full_models/Llama-2-13b-hf', tokenizer='/vllm/.idea/full_models/Llama-2-13b-hf', tokenizer_mode=auto, revision=None, tokenizer_revision=None, trust_remote_code=False, dtype=torch.float16, lora_config=None, delta_config=DeltaConfig(max_deltas=24, max_bitwidth=4, delta_dtype=torch.int8, max_cpu_deltas=32, delta_extra_vocab_size=0, pack_factor=Fraction(2, 1), kernel=<QuantKernel.TRITON: 'triton'>), swap_config=None, max_seq_len=4096, download_dir=None, load_format=auto, tensor_parallel_size=4, disable_custom_all_reduce=False, quantization=None, enforce_eager=True, kv_cache_dtype=auto, device_config=cuda, seed=0)
INFO 05-17 06:51:08 llm_engine.py:243] ignored verifying delta_config
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63dd20>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63d150>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63c880>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63e5f0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63eec0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63f790>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63e9e0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63e110>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63c3d0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63d390>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b2080a0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63fb20>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63d3f0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63c370>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63e140>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63f310>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63ea10>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63e800>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63df30>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63cee0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63fcd0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b208370>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63fe80>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63d510>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63c1f0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63e350>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63ec50>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63f7f0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63e6e0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63c580>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63d030>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63fd90>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63dcf0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63ffa0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63d000>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63c070>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63e650>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63e980>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b208070>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63c580>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63d030>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63fd90>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63dcf0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b617b20>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63d540>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63dc00>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63f880>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63cdc0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63dff0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b2089a0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63c2b0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63d270>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63fd30>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63fee0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b208400>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63dbd0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63dae0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63f850>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63d150>: Failed to establish a new connection: [Errno 111] Connection refused'))
Using bitblas
WARNING 05-17 07:00:50 models.py:48] Using bitblas delta modules
INFO 05-17 07:00:50 selector.py:16] Using FlashAttention backend.
[36m(RayWorkerVllm pid=98828)[0m Exception ignored in: <function TempDirectory.__del__ at 0x7f4bdf942200>
[36m(RayWorkerVllm pid=98828)[0m Traceback (most recent call last):
[36m(RayWorkerVllm pid=98828)[0m   File "/home/xiayao/.local/lib/python3.10/site-packages/bitblas/3rdparty/tvm/python/tvm/contrib/utils.py", line 145, in __del__
[36m(RayWorkerVllm pid=98828)[0m     self.remove()
[36m(RayWorkerVllm pid=98828)[0m   File "/home/xiayao/.local/lib/python3.10/site-packages/bitblas/3rdparty/tvm/python/tvm/contrib/utils.py", line 121, in remove
[36m(RayWorkerVllm pid=98828)[0m     if self.temp_dir:
[36m(RayWorkerVllm pid=98828)[0m AttributeError: 'TempDirectory' object has no attribute 'temp_dir'
[36m(RayWorkerVllm pid=98978)[0m Exception ignored in: <function TempDirectory.__del__ at 0x7f6a0594e200>
[36m(RayWorkerVllm pid=98978)[0m Traceback (most recent call last):
[36m(RayWorkerVllm pid=98978)[0m   File "/home/xiayao/.local/lib/python3.10/site-packages/bitblas/3rdparty/tvm/python/tvm/contrib/utils.py", line 145, in __del__
[36m(RayWorkerVllm pid=98978)[0m     self.remove()
[36m(RayWorkerVllm pid=98978)[0m   File "/home/xiayao/.local/lib/python3.10/site-packages/bitblas/3rdparty/tvm/python/tvm/contrib/utils.py", line 121, in remove
[36m(RayWorkerVllm pid=98978)[0m     if self.temp_dir:
[36m(RayWorkerVllm pid=98978)[0m AttributeError: 'TempDirectory' object has no attribute 'temp_dir'
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63c4f0>: Failed to establish a new connection: [Errno 111] Connection refused'))
[36m(RayWorkerVllm pid=98828)[0m Exception ignored in: <function TempDirectory.__del__ at 0x7f4bdf942200>
[36m(RayWorkerVllm pid=98828)[0m Traceback (most recent call last):
[36m(RayWorkerVllm pid=98828)[0m   File "/home/xiayao/.local/lib/python3.10/site-packages/bitblas/3rdparty/tvm/python/tvm/contrib/utils.py", line 145, in __del__
[36m(RayWorkerVllm pid=98828)[0m     self.remove()
[36m(RayWorkerVllm pid=98828)[0m   File "/home/xiayao/.local/lib/python3.10/site-packages/bitblas/3rdparty/tvm/python/tvm/contrib/utils.py", line 121, in remove
[36m(RayWorkerVllm pid=98828)[0m     if self.temp_dir:
[36m(RayWorkerVllm pid=98828)[0m AttributeError: 'TempDirectory' object has no attribute 'temp_dir'
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b208040>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63c160>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63d420>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63f910>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63da20>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b208be0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63fd00>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63d810>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63d4b0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63c220>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63e5c0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b209270>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63e530>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63c730>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63d120>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63dc30>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b208700>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63d540>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63da20>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63f910>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63d420>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63c490>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b208a60>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63e1a0>: Failed to establish a new connection: [Errno 111] Connection refused'))
[36m(RayWorkerVllm pid=98978)[0m Exception ignored in: <function TempDirectory.__del__ at 0x7f6a0594e200>
[36m(RayWorkerVllm pid=98978)[0m Traceback (most recent call last):
[36m(RayWorkerVllm pid=98978)[0m   File "/home/xiayao/.local/lib/python3.10/site-packages/bitblas/3rdparty/tvm/python/tvm/contrib/utils.py", line 145, in __del__
[36m(RayWorkerVllm pid=98978)[0m     self.remove()
[36m(RayWorkerVllm pid=98978)[0m   File "/home/xiayao/.local/lib/python3.10/site-packages/bitblas/3rdparty/tvm/python/tvm/contrib/utils.py", line 121, in remove
[36m(RayWorkerVllm pid=98978)[0m     if self.temp_dir:
[36m(RayWorkerVllm pid=98978)[0m AttributeError: 'TempDirectory' object has no attribute 'temp_dir'
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63e440>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63c130>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63d5d0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b2094e0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63d660>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63d1e0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63c640>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63e410>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63e1d0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b209b70>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63e500>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63cf70>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63fac0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63d0f0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63e830>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63e230>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63de40>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63e770>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63e380>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63cfd0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b2094b0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63f820>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63f970>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63d7b0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63c6a0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b617d60>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63e9e0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63ec50>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63db40>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63d210>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63c3a0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b209840>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63f9a0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63d600>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63d660>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63eb00>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b209960>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63e9e0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63ff40>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63d2a0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63ceb0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63f9d0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b209000>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63c250>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63d300>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63d6f0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63e9b0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b2096f0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63e800>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63ff40>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63d2a0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63ceb0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63d1b0>: Failed to establish a new connection: [Errno 111] Connection refused'))
[36m(RayWorkerVllm pid=98671)[0m Using bitblas
[36m(RayWorkerVllm pid=98828)[0m Using bitblas
[36m(RayWorkerVllm pid=98978)[0m Using bitblas
[36m(RayWorkerVllm pid=98828)[0m BitBLAS Operator not loaded, retrying in 5 seconds...
[36m(RayWorkerVllm pid=98828)[0m Error: [Errno 17] File exists: '/home/xiayao/.cache/bitblas/nvidia/nvidia-a100/c96df0f34eb23be516ef0126651d4e78b151337f28ebc87cc5492d79d135855f/tvm_rt_mod'
[36m(RayWorkerVllm pid=98978)[0m BitBLAS Operator not loaded, retrying in 5 seconds...
[36m(RayWorkerVllm pid=98978)[0m Error: [Errno 17] File exists: '/home/xiayao/.cache/bitblas/nvidia/nvidia-a100/c96df0f34eb23be516ef0126651d4e78b151337f28ebc87cc5492d79d135855f/tvm_rt_mod'
[36m(RayWorkerVllm pid=98828)[0m BitBLAS Operator not loaded, retrying in 5 seconds...
[36m(RayWorkerVllm pid=98828)[0m Error: [Errno 17] File exists: '/home/xiayao/.cache/bitblas/nvidia/nvidia-a100/c96df0f34eb23be516ef0126651d4e78b151337f28ebc87cc5492d79d135855f/tvm_rt_mod'
[36m(RayWorkerVllm pid=98978)[0m BitBLAS Operator not loaded, retrying in 5 seconds...
[36m(RayWorkerVllm pid=98978)[0m Error: [Errno 17] File exists: '/home/xiayao/.cache/bitblas/nvidia/nvidia-a100/3d178e04bf91ab34b5d0384b17a6649ba390c89bae6636085d64106067c19f7e/tvm_rt_mod'
[36m(RayWorkerVllm pid=98671)[0m WARNING 05-17 07:09:37 models.py:48] Using bitblas delta modules
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:09:37 selector.py:16] Using FlashAttention backend.
[36m(RayWorkerVllm pid=98828)[0m WARNING 05-17 07:09:46 models.py:48] Using bitblas delta modules
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:09:46 selector.py:16] Using FlashAttention backend.
[36m(RayWorkerVllm pid=98978)[0m WARNING 05-17 07:13:45 models.py:48] Using bitblas delta modules
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:13:45 selector.py:16] Using FlashAttention backend.
INFO 05-17 07:13:58 model_runner.py:161] Loading model weights took 6.1114 GB
INFO 05-17 07:13:58 worker_manager.py:297] Starting prefetching thread
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b2082e0>: Failed to establish a new connection: [Errno 111] Connection refused'))
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:14:00 model_runner.py:161] Loading model weights took 6.1114 GB
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:14:00 worker_manager.py:297] Starting prefetching thread
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:14:00 model_runner.py:161] Loading model weights took 6.1114 GB
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:14:00 worker_manager.py:297] Starting prefetching thread
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:14:00 model_runner.py:161] Loading model weights took 6.1114 GB
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:14:00 worker_manager.py:297] Starting prefetching thread
WARNING 05-17 07:14:02 quant_linear_triton.py:557] We should warm up the kernel here 
Waiting for 10 secs for the system to be ready: HTTPConnectionPool(host='localhost', port=8000): Max retries exceeded with url: /sysinfo (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f542b63c250>: Failed to establish a new connection: [Errno 111] Connection refused'))
[36m(RayWorkerVllm pid=98671)[0m WARNING 05-17 07:14:03 quant_linear_triton.py:557] We should warm up the kernel here 
[36m(RayWorkerVllm pid=98978)[0m WARNING 05-17 07:14:03 quant_linear_triton.py:557] We should warm up the kernel here 
[36m(RayWorkerVllm pid=98828)[0m WARNING 05-17 07:14:03 quant_linear_triton.py:557] We should warm up the kernel here 
INFO 05-17 07:14:16 ray_gpu_executor.py:275] # GPU blocks: 6431, # CPU blocks: 1310
INFO 05-17 07:14:19 scheduler.py:136] Using fcfs policy.
INFO 05-17 07:14:19 block_manager.py:264] disable automatic prefix caching
WARNING 05-17 07:14:19 serving_chat.py:373] No chat template provided. Chat API will not work.
INFO:     Started server process [90400]
INFO:     Waiting for application startup.
INFO:     Application startup complete.
INFO:     Uvicorn running on http://0.0.0.0:8000 (Press CTRL+C to quit)
INFO:     ::1:33584 - "GET /sysinfo HTTP/1.1" 200 OK
Translating from base-model to /vllm/.idea/full_models/Llama-2-13b-hf
Warming up starts
{'id': 56, 'prompt': 'USER: write code to generate answers to user input using ONNX\nASSISTANT:', 'timestamp': 0, 'model': 'delta-6', 'min_tokens': 327, 'max_tokens': 327}
INFO 05-17 07:14:20 worker_manager.py:332] Adding delta 6 to prefetching queue: [6]
WARNING 05-17 07:14:20 worker_manager.py:344] Missed prefetch, fall back to loading delta 6 now
INFO 05-17 07:14:20 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.0.safetensors']
INFO 05-17 07:14:20 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:14:20 worker_manager.py:332] Adding delta 6 to prefetching queue: [6]
[36m(RayWorkerVllm pid=98671)[0m WARNING 05-17 07:14:20 worker_manager.py:344] Missed prefetch, fall back to loading delta 6 now
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:14:20 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.1.safetensors']
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:14:20 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:14:20 worker_manager.py:332] Adding delta 6 to prefetching queue: [6]
[36m(RayWorkerVllm pid=98828)[0m WARNING 05-17 07:14:20 worker_manager.py:344] Missed prefetch, fall back to loading delta 6 now
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:14:20 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.2.safetensors']
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:14:20 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:14:20 worker_manager.py:332] Adding delta 6 to prefetching queue: [6]
[36m(RayWorkerVllm pid=98978)[0m WARNING 05-17 07:14:20 worker_manager.py:344] Missed prefetch, fall back to loading delta 6 now
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:14:20 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.3.safetensors']
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:14:20 models.py:263] [main] Lossless Compression Disabled
INFO 05-17 07:14:22 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 1.716 seconds
INFO 05-17 07:14:22 worker_manager.py:364] Main thread loading delta 6 done
INFO 05-17 07:14:22 worker_manager.py:379] [1715901262.5428078] activating delta
INFO 05-17 07:14:22 worker_manager.py:384] [1715901262.6525376] CPU -> GPU time: 0.1097
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:14:22 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.105 seconds
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:14:22 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.148 seconds
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:14:22 worker_manager.py:364] Main thread loading delta 6 done
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:14:22 worker_manager.py:379] [1715901262.9401085] activating delta
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:14:22 worker_manager.py:364] Main thread loading delta 6 done
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:14:22 worker_manager.py:379] [1715901262.897249] activating delta
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:14:22 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.173 seconds
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:14:23 worker_manager.py:384] [1715901263.0689375] CPU -> GPU time: 0.1288
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:14:23 worker_manager.py:384] [1715901263.0324597] CPU -> GPU time: 0.1352
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:14:22 worker_manager.py:364] Main thread loading delta 6 done
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:14:22 worker_manager.py:379] [1715901262.9737883] activating delta
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:14:23 worker_manager.py:384] [1715901263.115768] CPU -> GPU time: 0.1419
INFO 05-17 07:14:23 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 1 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-17 07:14:28 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 4.0 tokens/s, Running: 1 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-17 07:14:33 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 4.0 tokens/s, Running: 1 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.1%, CPU KV cache usage: 0.0%
INFO 05-17 07:14:38 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 4.1 tokens/s, Running: 1 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.1%, CPU KV cache usage: 0.0%
INFO 05-17 07:14:44 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 4.1 tokens/s, Running: 1 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.1%, CPU KV cache usage: 0.0%
INFO 05-17 07:14:49 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 4.1 tokens/s, Running: 1 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.1%, CPU KV cache usage: 0.0%
INFO 05-17 07:14:54 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 4.1 tokens/s, Running: 1 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.2%, CPU KV cache usage: 0.0%
INFO 05-17 07:14:59 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 4.1 tokens/s, Running: 1 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.2%, CPU KV cache usage: 0.0%
INFO 05-17 07:15:04 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 4.2 tokens/s, Running: 1 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.2%, CPU KV cache usage: 0.0%
INFO 05-17 07:15:09 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 4.1 tokens/s, Running: 1 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.2%, CPU KV cache usage: 0.0%
INFO 05-17 07:15:14 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 4.2 tokens/s, Running: 1 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.2%, CPU KV cache usage: 0.0%
INFO 05-17 07:15:19 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 4.1 tokens/s, Running: 1 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.2%, CPU KV cache usage: 0.0%
INFO 05-17 07:15:24 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 4.1 tokens/s, Running: 1 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.3%, CPU KV cache usage: 0.0%
INFO 05-17 07:15:29 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 4.1 tokens/s, Running: 1 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.3%, CPU KV cache usage: 0.0%
INFO 05-17 07:15:34 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 4.1 tokens/s, Running: 1 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.3%, CPU KV cache usage: 0.0%
INFO 05-17 07:15:39 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 4.1 tokens/s, Running: 1 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.3%, CPU KV cache usage: 0.0%
INFO:     ::1:33586 - "POST /v1/completions HTTP/1.1" 200 OK
Warming up ends
Issuing queries
sending 1 queries at 0.4
sending 1 queries at 1.1
sending 1 queries at 1.5
sending 1 queries at 1.9000000000000001
sending 1 queries at 2.2
sending 1 queries at 2.7
sending 1 queries at 3.0
sending 1 queries at 4.1000000000000005
sending 1 queries at 5.800000000000001
sending 1 queries at 6.0
sending 1 queries at 6.800000000000001
sending 1 queries at 7.2
sending 1 queries at 7.6000000000000005
sending 2 queries at 8.9
sending 2 queries at 9.0
sending 1 queries at 9.9
sending 1 queries at 10.600000000000001
sending 1 queries at 11.700000000000001
sending 1 queries at 13.600000000000001
sending 1 queries at 14.4
sending 1 queries at 14.700000000000001
sending 2 queries at 15.5
sending 1 queries at 16.0
sending 1 queries at 16.1
sending 1 queries at 17.6
sending 1 queries at 17.900000000000002
sending 1 queries at 18.2
sending 1 queries at 18.3
sending 1 queries at 19.1
sending 1 queries at 19.400000000000002
sending 2 queries at 19.8
sending 1 queries at 20.3
sending 1 queries at 20.8
sending 1 queries at 21.3
sending 1 queries at 22.700000000000003
sending 1 queries at 23.3
sending 1 queries at 23.5
sending 1 queries at 23.8
sending 2 queries at 24.400000000000002
sending 1 queries at 25.0
sending 1 queries at 25.5
sending 1 queries at 25.6
sending 1 queries at 25.700000000000003
sending 1 queries at 25.900000000000002
sending 1 queries at 26.1
sending 1 queries at 26.5
sending 1 queries at 26.8
sending 2 queries at 29.1
sending 1 queries at 29.200000000000003
sending 1 queries at 29.3
sending 1 queries at 29.8
sending 1 queries at 30.0
INFO 05-17 07:15:43 worker_manager.py:332] Adding delta 50 to prefetching queue: [50]
WARNING 05-17 07:15:43 worker_manager.py:344] Missed prefetch, fall back to loading delta 50 now
INFO 05-17 07:15:43 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.0.safetensors']
INFO 05-17 07:15:43 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:15:43 worker_manager.py:332] Adding delta 50 to prefetching queue: [50]
[36m(RayWorkerVllm pid=98671)[0m WARNING 05-17 07:15:43 worker_manager.py:344] Missed prefetch, fall back to loading delta 50 now
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:15:43 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.1.safetensors']
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:15:43 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:15:43 worker_manager.py:332] Adding delta 50 to prefetching queue: [50]
[36m(RayWorkerVllm pid=98828)[0m WARNING 05-17 07:15:43 worker_manager.py:344] Missed prefetch, fall back to loading delta 50 now
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:15:43 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.2.safetensors']
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:15:43 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:15:43 worker_manager.py:332] Adding delta 50 to prefetching queue: [50]
[36m(RayWorkerVllm pid=98978)[0m WARNING 05-17 07:15:43 worker_manager.py:344] Missed prefetch, fall back to loading delta 50 now
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:15:43 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.3.safetensors']
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:15:43 models.py:263] [main] Lossless Compression Disabled
INFO 05-17 07:15:43 worker_manager.py:332] Adding delta 5 to prefetching queue: [5]
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:15:56 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 13.399 seconds
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:15:56 worker_manager.py:364] Main thread loading delta 50 done
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:15:58 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 15.282 seconds
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:15:58 worker_manager.py:364] Main thread loading delta 50 done
INFO 05-17 07:15:59 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 16.250 seconds
INFO 05-17 07:15:59 worker_manager.py:364] Main thread loading delta 50 done
INFO 05-17 07:15:59 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.0.safetensors']
INFO 05-17 07:15:59 models.py:263] [prefetching] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:15:59 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 16.326 seconds
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:15:59 worker_manager.py:364] Main thread loading delta 50 done
INFO 05-17 07:16:01 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.6 tokens/s, Running: 1 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.0%, CPU KV cache usage: 0.0%
INFO 05-17 07:16:01 worker_manager.py:332] Adding delta 9 to prefetching queue: [5, 9]
INFO 05-17 07:16:01 worker_manager.py:332] Adding delta 1 to prefetching queue: [5, 9, 1]
INFO 05-17 07:16:01 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 1.559 seconds
INFO 05-17 07:16:01 worker_manager.py:278] Prefetching delta 5 done
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:16:01 worker_manager.py:332] Adding delta 5 to prefetching queue: [5]
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:16:01 worker_manager.py:332] Adding delta 9 to prefetching queue: [5, 9]
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:16:01 worker_manager.py:332] Adding delta 1 to prefetching queue: [5, 9, 1]
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:16:01 worker_manager.py:332] Adding delta 5 to prefetching queue: [5]
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:16:01 worker_manager.py:332] Adding delta 9 to prefetching queue: [5, 9]
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:16:01 worker_manager.py:332] Adding delta 1 to prefetching queue: [5, 9, 1]
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:16:01 worker_manager.py:332] Adding delta 5 to prefetching queue: [5]
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:16:01 worker_manager.py:332] Adding delta 9 to prefetching queue: [5, 9]
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:16:01 worker_manager.py:332] Adding delta 1 to prefetching queue: [5, 9, 1]
INFO 05-17 07:16:01 worker_manager.py:332] Adding delta 2 to prefetching queue: [9, 1, 2]
WARNING 05-17 07:16:01 worker_manager.py:344] Missed prefetch, fall back to loading delta 1 now
INFO 05-17 07:16:01 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.0.safetensors']
INFO 05-17 07:16:01 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98671)[0m WARNING 05-17 07:16:01 worker_manager.py:344] Missed prefetch, fall back to loading delta 1 now
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:16:01 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.1.safetensors']
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:16:01 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98828)[0m WARNING 05-17 07:16:01 worker_manager.py:344] Missed prefetch, fall back to loading delta 1 now
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:16:01 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.2.safetensors']
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:16:01 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98978)[0m WARNING 05-17 07:16:01 worker_manager.py:344] Missed prefetch, fall back to loading delta 1 now
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:16:01 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.3.safetensors']
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:16:01 models.py:263] [main] Lossless Compression Disabled
INFO 05-17 07:16:02 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 1.594 seconds
INFO 05-17 07:16:02 worker_manager.py:364] Main thread loading delta 1 done
INFO 05-17 07:16:02 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.0.safetensors']
INFO 05-17 07:16:02 models.py:263] [prefetching] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:16:03 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.081 seconds
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:16:03 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.041 seconds
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:16:03 worker_manager.py:364] Main thread loading delta 1 done
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:16:03 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.081 seconds
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:16:03 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.2.safetensors']
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:16:03 models.py:263] [prefetching] Lossless Compression Disabled
WARNING 05-17 07:16:03 worker_manager.py:344] Missed prefetch, fall back to loading delta 9 now
WARNING 05-17 07:16:03 worker_manager.py:353] Discarding prefetching delta 9
INFO 05-17 07:16:03 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.0.safetensors']
INFO 05-17 07:16:03 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:16:03 worker_manager.py:364] Main thread loading delta 1 done
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:16:03 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.1.safetensors']
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:16:03 models.py:263] [prefetching] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:16:03 worker_manager.py:364] Main thread loading delta 1 done
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:16:03 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.3.safetensors']
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:16:03 models.py:263] [prefetching] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98671)[0m WARNING 05-17 07:16:03 worker_manager.py:344] Missed prefetch, fall back to loading delta 5 now
[36m(RayWorkerVllm pid=98671)[0m WARNING 05-17 07:16:03 worker_manager.py:353] Discarding prefetching delta 5
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:16:03 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.1.safetensors']
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:16:03 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98828)[0m WARNING 05-17 07:16:03 worker_manager.py:344] Missed prefetch, fall back to loading delta 5 now
[36m(RayWorkerVllm pid=98828)[0m WARNING 05-17 07:16:03 worker_manager.py:353] Discarding prefetching delta 5
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:16:03 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.2.safetensors']
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:16:03 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98978)[0m WARNING 05-17 07:16:03 worker_manager.py:344] Missed prefetch, fall back to loading delta 5 now
[36m(RayWorkerVllm pid=98978)[0m WARNING 05-17 07:16:03 worker_manager.py:353] Discarding prefetching delta 5
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:16:03 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.3.safetensors']
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:16:03 models.py:263] [main] Lossless Compression Disabled
INFO 05-17 07:16:05 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.662 seconds
INFO 05-17 07:16:05 worker_manager.py:278] Prefetching delta 9 done
INFO 05-17 07:16:06 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.832 seconds
INFO 05-17 07:16:06 worker_manager.py:364] Main thread loading delta 9 done
INFO 05-17 07:16:06 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.0.safetensors']
INFO 05-17 07:16:06 models.py:263] [prefetching] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:16:06 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 3.172 seconds
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:16:06 worker_manager.py:278] Prefetching delta 5 done
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:16:06 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 3.227 seconds
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:16:06 worker_manager.py:278] Prefetching delta 5 done
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:16:06 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 3.345 seconds
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:16:06 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 3.252 seconds
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:16:06 worker_manager.py:364] Main thread loading delta 5 done
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:16:06 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.1.safetensors']
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:16:06 models.py:263] [prefetching] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:16:06 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 3.306 seconds
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:16:06 worker_manager.py:278] Prefetching delta 5 done
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:16:06 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 3.270 seconds
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:16:06 worker_manager.py:364] Main thread loading delta 5 done
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:16:06 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.2.safetensors']
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:16:06 models.py:263] [prefetching] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:16:06 worker_manager.py:364] Main thread loading delta 5 done
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:16:06 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.3.safetensors']
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:16:06 models.py:263] [prefetching] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98671)[0m WARNING 05-17 07:16:07 worker_manager.py:344] Missed prefetch, fall back to loading delta 9 now
[36m(RayWorkerVllm pid=98671)[0m WARNING 05-17 07:16:07 worker_manager.py:353] Discarding prefetching delta 9
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:16:07 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.1.safetensors']
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:16:07 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98978)[0m WARNING 05-17 07:16:07 worker_manager.py:344] Missed prefetch, fall back to loading delta 9 now
[36m(RayWorkerVllm pid=98978)[0m WARNING 05-17 07:16:07 worker_manager.py:353] Discarding prefetching delta 9
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:16:07 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.3.safetensors']
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:16:07 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98828)[0m WARNING 05-17 07:16:07 worker_manager.py:344] Missed prefetch, fall back to loading delta 9 now
[36m(RayWorkerVllm pid=98828)[0m WARNING 05-17 07:16:07 worker_manager.py:353] Discarding prefetching delta 9
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:16:07 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.2.safetensors']
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:16:07 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:16:08 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 1.963 seconds
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:16:08 worker_manager.py:278] Prefetching delta 9 done
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:16:08 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.051 seconds
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:16:09 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 1.973 seconds
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:16:09 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.069 seconds
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:16:09 worker_manager.py:278] Prefetching delta 9 done
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:16:09 worker_manager.py:278] Prefetching delta 9 done
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:16:09 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 1.963 seconds
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:16:09 worker_manager.py:364] Main thread loading delta 9 done
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:16:09 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.065 seconds
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:16:09 worker_manager.py:364] Main thread loading delta 9 done
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:16:09 worker_manager.py:364] Main thread loading delta 9 done
INFO 05-17 07:16:09 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 3.121 seconds
INFO 05-17 07:16:09 worker_manager.py:278] Prefetching delta 2 done
INFO 05-17 07:16:10 worker_manager.py:332] Adding delta 7 to prefetching queue: [7]
INFO 05-17 07:16:10 worker_manager.py:332] Adding delta 4 to prefetching queue: [7, 4]
INFO 05-17 07:16:10 metrics.py:276] Avg prompt throughput: 19.5 tokens/s, Avg generation throughput: 1.0 tokens/s, Running: 9 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.3%, CPU KV cache usage: 0.0%
INFO 05-17 07:16:10 worker_manager.py:332] Adding delta 3 to prefetching queue: [7, 4, 3]
INFO 05-17 07:16:10 worker_manager.py:332] Adding delta 32 to prefetching queue: [7, 4, 3, 32]
WARNING 05-17 07:16:10 worker_manager.py:344] Missed prefetch, fall back to loading delta 32 now
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:16:10 worker_manager.py:332] Adding delta 2 to prefetching queue: [2]
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:16:10 worker_manager.py:332] Adding delta 7 to prefetching queue: [2, 7]
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:16:10 worker_manager.py:332] Adding delta 4 to prefetching queue: [2, 7, 4]
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:16:10 worker_manager.py:332] Adding delta 3 to prefetching queue: [2, 7, 4, 3]
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:16:10 worker_manager.py:332] Adding delta 32 to prefetching queue: [2, 7, 4, 3, 32]
[36m(RayWorkerVllm pid=98671)[0m WARNING 05-17 07:16:10 worker_manager.py:344] Missed prefetch, fall back to loading delta 32 now
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:16:10 worker_manager.py:332] Adding delta 2 to prefetching queue: [2]
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:16:10 worker_manager.py:332] Adding delta 7 to prefetching queue: [2, 7]
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:16:10 worker_manager.py:332] Adding delta 4 to prefetching queue: [2, 7, 4]
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:16:10 worker_manager.py:332] Adding delta 3 to prefetching queue: [2, 7, 4, 3]
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:16:10 worker_manager.py:332] Adding delta 32 to prefetching queue: [2, 7, 4, 3, 32]
[36m(RayWorkerVllm pid=98828)[0m WARNING 05-17 07:16:10 worker_manager.py:344] Missed prefetch, fall back to loading delta 32 now
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:16:10 worker_manager.py:332] Adding delta 2 to prefetching queue: [2]
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:16:10 worker_manager.py:332] Adding delta 7 to prefetching queue: [2, 7]
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:16:10 worker_manager.py:332] Adding delta 4 to prefetching queue: [2, 7, 4]
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:16:10 worker_manager.py:332] Adding delta 3 to prefetching queue: [2, 7, 4, 3]
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:16:10 worker_manager.py:332] Adding delta 32 to prefetching queue: [2, 7, 4, 3, 32]
[36m(RayWorkerVllm pid=98978)[0m WARNING 05-17 07:16:10 worker_manager.py:344] Missed prefetch, fall back to loading delta 32 now
INFO 05-17 07:16:10 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.0.safetensors']
INFO 05-17 07:16:10 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:16:10 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.1.safetensors']
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:16:10 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:16:10 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.2.safetensors']
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:16:10 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:16:10 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.3.safetensors']
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:16:10 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:16:11 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 0.595 seconds
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:16:11 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 0.624 seconds
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:16:11 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 0.641 seconds
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:16:11 worker_manager.py:364] Main thread loading delta 32 done
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:16:11 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.1.safetensors']
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:16:11 models.py:263] [prefetching] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:16:11 worker_manager.py:364] Main thread loading delta 32 done
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:16:11 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.2.safetensors']
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:16:11 models.py:263] [prefetching] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:16:11 worker_manager.py:364] Main thread loading delta 32 done
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:16:11 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.3.safetensors']
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:16:11 models.py:263] [prefetching] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98828)[0m WARNING 05-17 07:16:11 worker_manager.py:344] Missed prefetch, fall back to loading delta 2 now
[36m(RayWorkerVllm pid=98828)[0m WARNING 05-17 07:16:11 worker_manager.py:353] Discarding prefetching delta 2
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:16:11 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.2.safetensors']
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:16:11 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98671)[0m WARNING 05-17 07:16:11 worker_manager.py:344] Missed prefetch, fall back to loading delta 2 now
[36m(RayWorkerVllm pid=98671)[0m WARNING 05-17 07:16:11 worker_manager.py:353] Discarding prefetching delta 2
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:16:11 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.1.safetensors']
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:16:11 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98978)[0m WARNING 05-17 07:16:11 worker_manager.py:344] Missed prefetch, fall back to loading delta 2 now
[36m(RayWorkerVllm pid=98978)[0m WARNING 05-17 07:16:11 worker_manager.py:353] Discarding prefetching delta 2
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:16:11 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.3.safetensors']
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:16:11 models.py:263] [main] Lossless Compression Disabled
INFO 05-17 07:16:11 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 1.438 seconds
INFO 05-17 07:16:11 worker_manager.py:364] Main thread loading delta 32 done
INFO 05-17 07:16:11 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.0.safetensors']
INFO 05-17 07:16:11 models.py:263] [prefetching] Lossless Compression Disabled
WARNING 05-17 07:16:12 worker_manager.py:344] Missed prefetch, fall back to loading delta 3 now
INFO 05-17 07:16:12 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.0.safetensors']
INFO 05-17 07:16:12 models.py:263] [main] Lossless Compression Disabled
total threads: 58
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:16:14 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 3.114 seconds
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:16:14 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 3.147 seconds
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:16:14 worker_manager.py:278] Prefetching delta 2 done
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:16:14 worker_manager.py:278] Prefetching delta 2 done
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:16:14 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 3.243 seconds
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:16:14 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 3.197 seconds
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:16:14 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 3.194 seconds
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:16:14 worker_manager.py:364] Main thread loading delta 2 done
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:16:14 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.2.safetensors']
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:16:14 models.py:263] [prefetching] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:16:14 worker_manager.py:278] Prefetching delta 2 done
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:16:14 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 3.161 seconds
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:16:14 worker_manager.py:364] Main thread loading delta 2 done
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:16:14 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.3.safetensors']
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:16:14 models.py:263] [prefetching] Lossless Compression Disabled
INFO 05-17 07:16:14 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.667 seconds
INFO 05-17 07:16:14 worker_manager.py:278] Prefetching delta 7 done
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:16:14 worker_manager.py:364] Main thread loading delta 2 done
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:16:14 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.1.safetensors']
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:16:14 models.py:263] [prefetching] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98671)[0m WARNING 05-17 07:16:14 worker_manager.py:344] Missed prefetch, fall back to loading delta 3 now
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:16:14 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.1.safetensors']
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:16:14 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98828)[0m WARNING 05-17 07:16:14 worker_manager.py:344] Missed prefetch, fall back to loading delta 3 now
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:16:14 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.2.safetensors']
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:16:14 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98978)[0m WARNING 05-17 07:16:14 worker_manager.py:344] Missed prefetch, fall back to loading delta 3 now
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:16:14 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.3.safetensors']
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:16:14 models.py:263] [main] Lossless Compression Disabled
INFO 05-17 07:16:15 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.774 seconds
INFO 05-17 07:16:15 worker_manager.py:364] Main thread loading delta 3 done
INFO 05-17 07:16:15 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.0.safetensors']
INFO 05-17 07:16:15 models.py:263] [prefetching] Lossless Compression Disabled
WARNING 05-17 07:16:15 worker_manager.py:344] Missed prefetch, fall back to loading delta 4 now
WARNING 05-17 07:16:15 worker_manager.py:353] Discarding prefetching delta 4
INFO 05-17 07:16:15 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.0.safetensors']
INFO 05-17 07:16:15 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:16:16 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 1.876 seconds
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:16:16 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 1.863 seconds
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:16:16 worker_manager.py:278] Prefetching delta 7 done
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:16:16 worker_manager.py:278] Prefetching delta 7 done
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:16:16 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 1.927 seconds
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:16:16 worker_manager.py:278] Prefetching delta 7 done
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:16:16 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.104 seconds
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:16:16 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.116 seconds
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:16:16 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.046 seconds
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:16:16 worker_manager.py:364] Main thread loading delta 3 done
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:16:16 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.1.safetensors']
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:16:16 models.py:263] [prefetching] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:16:16 worker_manager.py:364] Main thread loading delta 3 done
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:16:16 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.2.safetensors']
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:16:16 models.py:263] [prefetching] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:16:16 worker_manager.py:364] Main thread loading delta 3 done
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:16:16 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.3.safetensors']
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:16:16 models.py:263] [prefetching] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98671)[0m WARNING 05-17 07:16:17 worker_manager.py:344] Missed prefetch, fall back to loading delta 4 now
[36m(RayWorkerVllm pid=98671)[0m WARNING 05-17 07:16:17 worker_manager.py:353] Discarding prefetching delta 4
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:16:17 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.1.safetensors']
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:16:17 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98828)[0m WARNING 05-17 07:16:17 worker_manager.py:344] Missed prefetch, fall back to loading delta 4 now
[36m(RayWorkerVllm pid=98828)[0m WARNING 05-17 07:16:17 worker_manager.py:353] Discarding prefetching delta 4
[36m(RayWorkerVllm pid=98978)[0m WARNING 05-17 07:16:16 worker_manager.py:344] Missed prefetch, fall back to loading delta 4 now
[36m(RayWorkerVllm pid=98978)[0m WARNING 05-17 07:16:16 worker_manager.py:353] Discarding prefetching delta 4
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:16:17 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.3.safetensors']
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:16:17 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:16:17 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.2.safetensors']
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:16:17 models.py:263] [main] Lossless Compression Disabled
INFO 05-17 07:16:18 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.895 seconds
INFO 05-17 07:16:18 worker_manager.py:278] Prefetching delta 4 done
INFO 05-17 07:16:18 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 3.007 seconds
INFO 05-17 07:16:18 worker_manager.py:364] Main thread loading delta 4 done
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:16:19 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.988 seconds
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:16:19 worker_manager.py:278] Prefetching delta 4 done
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:16:19 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 3.025 seconds
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:16:19 worker_manager.py:278] Prefetching delta 4 done
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:16:19 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 3.126 seconds
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:16:20 worker_manager.py:278] Prefetching delta 4 done
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:16:20 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 3.041 seconds
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:16:20 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 3.088 seconds
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:16:20 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 3.098 seconds
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:16:20 worker_manager.py:364] Main thread loading delta 4 done
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:16:20 worker_manager.py:364] Main thread loading delta 4 done
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:16:20 worker_manager.py:364] Main thread loading delta 4 done
INFO 05-17 07:16:21 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 18 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.6%, CPU KV cache usage: 0.0%
INFO 05-17 07:16:21 worker_manager.py:332] Adding delta 8 to prefetching queue: [8]
INFO 05-17 07:16:21 worker_manager.py:332] Adding delta 14 to prefetching queue: [8, 14]
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:16:21 worker_manager.py:332] Adding delta 8 to prefetching queue: [8]
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:16:21 worker_manager.py:332] Adding delta 14 to prefetching queue: [8, 14]
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:16:21 worker_manager.py:332] Adding delta 8 to prefetching queue: [8]
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:16:21 worker_manager.py:332] Adding delta 14 to prefetching queue: [8, 14]
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:16:21 worker_manager.py:332] Adding delta 8 to prefetching queue: [8]
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:16:21 worker_manager.py:332] Adding delta 14 to prefetching queue: [8, 14]
WARNING 05-17 07:16:21 worker_manager.py:344] Missed prefetch, fall back to loading delta 8 now
INFO 05-17 07:16:21 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.0.safetensors']
INFO 05-17 07:16:21 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98671)[0m WARNING 05-17 07:16:21 worker_manager.py:344] Missed prefetch, fall back to loading delta 8 now
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:16:21 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.1.safetensors']
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:16:21 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98828)[0m WARNING 05-17 07:16:21 worker_manager.py:344] Missed prefetch, fall back to loading delta 8 now
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:16:21 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.2.safetensors']
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:16:21 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98978)[0m WARNING 05-17 07:16:21 worker_manager.py:344] Missed prefetch, fall back to loading delta 8 now
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:16:21 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.3.safetensors']
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:16:21 models.py:263] [main] Lossless Compression Disabled
INFO 05-17 07:16:22 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 0.160 seconds
INFO 05-17 07:16:22 worker_manager.py:364] Main thread loading delta 8 done
WARNING 05-17 07:16:22 worker_manager.py:344] Missed prefetch, fall back to loading delta 14 now
INFO 05-17 07:16:22 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.0.safetensors']
INFO 05-17 07:16:22 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:16:22 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 0.666 seconds
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:16:22 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 0.700 seconds
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:16:22 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 0.697 seconds
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:16:22 worker_manager.py:364] Main thread loading delta 8 done
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:16:22 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.1.safetensors']
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:16:22 models.py:263] [prefetching] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:16:22 worker_manager.py:364] Main thread loading delta 8 done
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:16:22 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.2.safetensors']
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:16:22 models.py:263] [prefetching] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:16:22 worker_manager.py:364] Main thread loading delta 8 done
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:16:22 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.3.safetensors']
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:16:22 models.py:263] [prefetching] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98671)[0m WARNING 05-17 07:16:22 worker_manager.py:344] Missed prefetch, fall back to loading delta 14 now
[36m(RayWorkerVllm pid=98671)[0m WARNING 05-17 07:16:22 worker_manager.py:353] Discarding prefetching delta 14
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:16:22 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.1.safetensors']
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:16:22 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98828)[0m WARNING 05-17 07:16:22 worker_manager.py:344] Missed prefetch, fall back to loading delta 14 now
[36m(RayWorkerVllm pid=98828)[0m WARNING 05-17 07:16:22 worker_manager.py:353] Discarding prefetching delta 14
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:16:22 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.2.safetensors']
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:16:22 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98978)[0m WARNING 05-17 07:16:22 worker_manager.py:344] Missed prefetch, fall back to loading delta 14 now
[36m(RayWorkerVllm pid=98978)[0m WARNING 05-17 07:16:22 worker_manager.py:353] Discarding prefetching delta 14
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:16:22 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.3.safetensors']
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:16:22 models.py:263] [main] Lossless Compression Disabled
INFO 05-17 07:16:23 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 1.627 seconds
INFO 05-17 07:16:23 worker_manager.py:364] Main thread loading delta 14 done
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:16:25 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 3.088 seconds
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:16:25 worker_manager.py:278] Prefetching delta 14 done
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:16:25 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 3.072 seconds
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:16:25 worker_manager.py:278] Prefetching delta 14 done
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:16:25 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 3.144 seconds
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:16:25 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 3.099 seconds
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:16:25 worker_manager.py:278] Prefetching delta 14 done
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:16:25 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 3.052 seconds
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:16:25 worker_manager.py:364] Main thread loading delta 14 done
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:16:25 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 3.119 seconds
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:16:25 worker_manager.py:364] Main thread loading delta 14 done
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:16:26 worker_manager.py:364] Main thread loading delta 14 done
INFO:     ::1:33876 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 07:16:27 worker_manager.py:332] Adding delta 43 to prefetching queue: [43]
INFO 05-17 07:16:27 metrics.py:276] Avg prompt throughput: 94.3 tokens/s, Avg generation throughput: 3.2 tokens/s, Running: 26 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.9%, CPU KV cache usage: 0.0%
INFO 05-17 07:16:27 worker_manager.py:332] Adding delta 40 to prefetching queue: [43, 40]
WARNING 05-17 07:16:27 worker_manager.py:344] Missed prefetch, fall back to loading delta 43 now
INFO 05-17 07:16:27 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.0.safetensors']
INFO 05-17 07:16:27 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:16:27 worker_manager.py:332] Adding delta 43 to prefetching queue: [43]
[36m(RayWorkerVllm pid=98671)[0m WARNING 05-17 07:16:27 worker_manager.py:344] Missed prefetch, fall back to loading delta 43 now
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:16:27 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.1.safetensors']
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:16:27 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:16:27 worker_manager.py:332] Adding delta 43 to prefetching queue: [43]
[36m(RayWorkerVllm pid=98828)[0m WARNING 05-17 07:16:27 worker_manager.py:344] Missed prefetch, fall back to loading delta 43 now
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:16:27 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.2.safetensors']
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:16:27 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:16:27 worker_manager.py:332] Adding delta 43 to prefetching queue: [43]
[36m(RayWorkerVllm pid=98978)[0m WARNING 05-17 07:16:27 worker_manager.py:344] Missed prefetch, fall back to loading delta 43 now
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:16:27 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.3.safetensors']
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:16:27 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:16:37 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 10.112 seconds
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:16:37 worker_manager.py:364] Main thread loading delta 43 done
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:16:40 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 13.198 seconds
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:16:40 worker_manager.py:364] Main thread loading delta 43 done
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:16:42 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 14.898 seconds
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:16:42 worker_manager.py:364] Main thread loading delta 43 done
INFO 05-17 07:16:42 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 15.269 seconds
INFO 05-17 07:16:42 worker_manager.py:364] Main thread loading delta 43 done
INFO 05-17 07:16:42 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.0.safetensors']
INFO 05-17 07:16:42 models.py:263] [prefetching] Lossless Compression Disabled
INFO 05-17 07:16:45 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 0.0 tokens/s, Running: 35 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 1.2%, CPU KV cache usage: 0.0%
INFO 05-17 07:16:45 worker_manager.py:332] Adding delta 16 to prefetching queue: [40, 16]
INFO 05-17 07:16:45 worker_manager.py:332] Adding delta 13 to prefetching queue: [40, 16, 13]
INFO 05-17 07:16:45 worker_manager.py:332] Adding delta 20 to prefetching queue: [40, 16, 13, 20]
WARNING 05-17 07:16:45 worker_manager.py:344] Missed prefetch, fall back to loading delta 40 now
WARNING 05-17 07:16:45 worker_manager.py:353] Discarding prefetching delta 40
INFO 05-17 07:16:45 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.0.safetensors']
INFO 05-17 07:16:45 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:16:45 worker_manager.py:332] Adding delta 40 to prefetching queue: [40]
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:16:45 worker_manager.py:332] Adding delta 16 to prefetching queue: [40, 16]
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:16:45 worker_manager.py:332] Adding delta 13 to prefetching queue: [40, 16, 13]
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:16:45 worker_manager.py:332] Adding delta 20 to prefetching queue: [40, 16, 13, 20]
[36m(RayWorkerVllm pid=98671)[0m WARNING 05-17 07:16:45 worker_manager.py:344] Missed prefetch, fall back to loading delta 40 now
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:16:45 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.1.safetensors']
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:16:45 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:16:45 worker_manager.py:332] Adding delta 40 to prefetching queue: [40]
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:16:45 worker_manager.py:332] Adding delta 16 to prefetching queue: [40, 16]
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:16:45 worker_manager.py:332] Adding delta 13 to prefetching queue: [40, 16, 13]
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:16:45 worker_manager.py:332] Adding delta 20 to prefetching queue: [40, 16, 13, 20]
[36m(RayWorkerVllm pid=98828)[0m WARNING 05-17 07:16:45 worker_manager.py:344] Missed prefetch, fall back to loading delta 40 now
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:16:45 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.2.safetensors']
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:16:45 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:16:45 worker_manager.py:332] Adding delta 40 to prefetching queue: [40]
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:16:45 worker_manager.py:332] Adding delta 16 to prefetching queue: [40, 16]
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:16:45 worker_manager.py:332] Adding delta 13 to prefetching queue: [40, 16, 13]
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:16:45 worker_manager.py:332] Adding delta 20 to prefetching queue: [40, 16, 13, 20]
[36m(RayWorkerVllm pid=98978)[0m WARNING 05-17 07:16:45 worker_manager.py:344] Missed prefetch, fall back to loading delta 40 now
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:16:45 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.3.safetensors']
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:16:45 models.py:263] [main] Lossless Compression Disabled
INFO 05-17 07:16:57 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 11.726 seconds
INFO 05-17 07:16:57 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 14.232 seconds
INFO 05-17 07:16:57 worker_manager.py:364] Main thread loading delta 40 done
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:16:57 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 11.772 seconds
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:16:57 worker_manager.py:364] Main thread loading delta 40 done
INFO 05-17 07:16:57 worker_manager.py:278] Prefetching delta 40 done
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:16:57 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.2.safetensors']
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:16:57 models.py:263] [prefetching] Lossless Compression Disabled
WARNING 05-17 07:16:57 worker_manager.py:344] Missed prefetch, fall back to loading delta 13 now
INFO 05-17 07:16:57 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.0.safetensors']
INFO 05-17 07:16:57 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98828)[0m WARNING 05-17 07:16:57 worker_manager.py:344] Missed prefetch, fall back to loading delta 13 now
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:16:57 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.2.safetensors']
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:16:57 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:16:58 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 13.615 seconds
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:16:59 worker_manager.py:364] Main thread loading delta 40 done
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:16:59 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.3.safetensors']
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:16:59 models.py:263] [prefetching] Lossless Compression Disabled
INFO 05-17 07:16:59 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 1.753 seconds
[36m(RayWorkerVllm pid=98978)[0m WARNING 05-17 07:16:59 worker_manager.py:344] Missed prefetch, fall back to loading delta 13 now
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:16:59 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.3.safetensors']
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:16:59 models.py:263] [main] Lossless Compression Disabled
INFO 05-17 07:16:59 worker_manager.py:364] Main thread loading delta 13 done
INFO 05-17 07:16:59 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.0.safetensors']
INFO 05-17 07:16:59 models.py:263] [prefetching] Lossless Compression Disabled
WARNING 05-17 07:16:59 worker_manager.py:344] Missed prefetch, fall back to loading delta 16 now
WARNING 05-17 07:16:59 worker_manager.py:353] Discarding prefetching delta 16
INFO 05-17 07:16:59 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.0.safetensors']
INFO 05-17 07:16:59 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:17:00 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 15.107 seconds
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:17:00 worker_manager.py:364] Main thread loading delta 40 done
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:17:00 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.1.safetensors']
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:17:00 models.py:263] [prefetching] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:17:00 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 3.297 seconds
[36m(RayWorkerVllm pid=98671)[0m WARNING 05-17 07:17:00 worker_manager.py:344] Missed prefetch, fall back to loading delta 13 now
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:17:00 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.1.safetensors']
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:17:00 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:17:00 worker_manager.py:278] Prefetching delta 16 done
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:17:00 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 3.320 seconds
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:17:00 worker_manager.py:364] Main thread loading delta 13 done
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:17:00 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.2.safetensors']
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:17:00 models.py:263] [prefetching] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98828)[0m WARNING 05-17 07:17:01 worker_manager.py:344] Missed prefetch, fall back to loading delta 20 now
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:17:01 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.2.safetensors']
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:17:01 models.py:263] [main] Lossless Compression Disabled
INFO 05-17 07:17:02 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.973 seconds
INFO 05-17 07:17:02 worker_manager.py:278] Prefetching delta 16 done
INFO 05-17 07:17:02 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 2.821 seconds
INFO 05-17 07:17:02 worker_manager.py:364] Main thread loading delta 16 done
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:17:02 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 3.216 seconds
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:17:02 worker_manager.py:278] Prefetching delta 16 done
INFO 05-17 07:17:02 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.0.safetensors']
INFO 05-17 07:17:02 models.py:263] [prefetching] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:17:02 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 3.177 seconds
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:17:02 worker_manager.py:364] Main thread loading delta 13 done
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:17:02 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.3.safetensors']
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:17:02 models.py:263] [prefetching] Lossless Compression Disabled
WARNING 05-17 07:17:02 worker_manager.py:344] Missed prefetch, fall back to loading delta 20 now
WARNING 05-17 07:17:02 worker_manager.py:353] Discarding prefetching delta 20
INFO 05-17 07:17:02 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.0.safetensors']
INFO 05-17 07:17:02 models.py:263] [main] Lossless Compression Disabled
INFO 05-17 07:17:02 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 0.223 seconds
INFO 05-17 07:17:02 worker_manager.py:278] Prefetching delta 20 done
[36m(RayWorkerVllm pid=98978)[0m WARNING 05-17 07:17:02 worker_manager.py:344] Missed prefetch, fall back to loading delta 20 now
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:17:02 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.3.safetensors']
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:17:02 models.py:263] [main] Lossless Compression Disabled
INFO 05-17 07:17:03 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 0.583 seconds
INFO 05-17 07:17:03 worker_manager.py:364] Main thread loading delta 20 done
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:17:03 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 3.353 seconds
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:17:03 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 3.045 seconds
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:17:03 worker_manager.py:278] Prefetching delta 13 done
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:17:04 worker_manager.py:278] Prefetching delta 16 done
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:17:04 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 3.466 seconds
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:17:04 worker_manager.py:364] Main thread loading delta 13 done
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:17:04 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.1.safetensors']
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:17:04 models.py:263] [prefetching] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:17:04 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 3.156 seconds
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:17:04 worker_manager.py:364] Main thread loading delta 20 done
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:17:04 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.2.safetensors']
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:17:04 models.py:263] [prefetching] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98671)[0m WARNING 05-17 07:17:04 worker_manager.py:344] Missed prefetch, fall back to loading delta 20 now
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:17:04 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.1.safetensors']
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:17:04 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:17:05 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 3.451 seconds
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:17:05 worker_manager.py:278] Prefetching delta 13 done
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:17:06 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 3.477 seconds
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:17:06 worker_manager.py:364] Main thread loading delta 20 done
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:17:06 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.3.safetensors']
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:17:06 models.py:263] [prefetching] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:17:07 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 3.187 seconds
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:17:07 worker_manager.py:278] Prefetching delta 13 done
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:17:07 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 3.239 seconds
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:17:07 worker_manager.py:364] Main thread loading delta 20 done
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:17:07 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.1.safetensors']
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:17:07 models.py:263] [prefetching] Lossless Compression Disabled
INFO 05-17 07:17:09 metrics.py:276] Avg prompt throughput: 10.4 tokens/s, Avg generation throughput: 0.4 tokens/s, Running: 44 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 1.5%, CPU KV cache usage: 0.0%
INFO 05-17 07:17:09 worker_manager.py:332] Adding delta 41 to prefetching queue: [41]
WARNING 05-17 07:17:09 worker_manager.py:344] Missed prefetch, fall back to loading delta 41 now
INFO 05-17 07:17:09 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.0.safetensors']
INFO 05-17 07:17:09 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:17:09 worker_manager.py:332] Adding delta 41 to prefetching queue: [41]
[36m(RayWorkerVllm pid=98671)[0m WARNING 05-17 07:17:09 worker_manager.py:344] Missed prefetch, fall back to loading delta 41 now
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:17:09 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.1.safetensors']
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:17:09 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:17:09 worker_manager.py:332] Adding delta 41 to prefetching queue: [41]
[36m(RayWorkerVllm pid=98828)[0m WARNING 05-17 07:17:09 worker_manager.py:344] Missed prefetch, fall back to loading delta 41 now
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:17:09 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.2.safetensors']
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:17:09 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:17:09 worker_manager.py:332] Adding delta 41 to prefetching queue: [41]
[36m(RayWorkerVllm pid=98978)[0m WARNING 05-17 07:17:09 worker_manager.py:344] Missed prefetch, fall back to loading delta 41 now
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:17:09 models.py:210] Loading from ['bitblas.remain.safetensors', 'bitblas.rank.3.safetensors']
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:17:09 models.py:263] [main] Lossless Compression Disabled
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:17:10 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 6.543 seconds
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:17:10 worker_manager.py:278] Prefetching delta 20 done
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:17:10 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 3.223 seconds
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:17:11 worker_manager.py:278] Prefetching delta 20 done
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:17:11 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 4.806 seconds
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:17:11 worker_manager.py:278] Prefetching delta 20 done
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:17:19 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 10.255 seconds
[36m(RayWorkerVllm pid=98671)[0m INFO 05-17 07:17:19 worker_manager.py:364] Main thread loading delta 41 done
INFO 05-17 07:17:22 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 13.106 seconds
INFO 05-17 07:17:22 worker_manager.py:364] Main thread loading delta 41 done
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:17:23 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 14.274 seconds
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:17:23 models.py:321] Disk -> CPU: Loaded 1674.62 MiB in 14.330 seconds
[36m(RayWorkerVllm pid=98978)[0m INFO 05-17 07:17:23 worker_manager.py:364] Main thread loading delta 41 done
[36m(RayWorkerVllm pid=98828)[0m INFO 05-17 07:17:23 worker_manager.py:364] Main thread loading delta 41 done
INFO 05-17 07:17:26 metrics.py:276] Avg prompt throughput: 12.6 tokens/s, Avg generation throughput: 0.5 tokens/s, Running: 53 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 1.8%, CPU KV cache usage: 0.0%
INFO:     ::1:33920 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 07:17:31 metrics.py:276] Avg prompt throughput: 77.8 tokens/s, Avg generation throughput: 21.4 tokens/s, Running: 56 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 2.1%, CPU KV cache usage: 0.0%
INFO 05-17 07:17:37 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 21.8 tokens/s, Running: 54 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 2.1%, CPU KV cache usage: 0.0%
INFO:     ::1:33866 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:33908 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:33930 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:33934 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:33928 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 07:17:42 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 20.9 tokens/s, Running: 50 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 2.0%, CPU KV cache usage: 0.0%
INFO:     ::1:33926 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:33952 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 07:17:49 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 20.4 tokens/s, Running: 49 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 2.1%, CPU KV cache usage: 0.0%
INFO:     ::1:33932 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:33862 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 07:17:56 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 14.6 tokens/s, Running: 47 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 2.2%, CPU KV cache usage: 0.0%
INFO 05-17 07:18:01 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 25.0 tokens/s, Running: 47 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 2.3%, CPU KV cache usage: 0.0%
INFO 05-17 07:18:09 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 19.1 tokens/s, Running: 47 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 2.5%, CPU KV cache usage: 0.0%
INFO 05-17 07:18:16 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 13.5 tokens/s, Running: 47 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 2.6%, CPU KV cache usage: 0.0%
INFO 05-17 07:18:21 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 26.9 tokens/s, Running: 47 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 2.6%, CPU KV cache usage: 0.0%
INFO:     ::1:33970 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 07:18:28 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 19.1 tokens/s, Running: 46 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 2.7%, CPU KV cache usage: 0.0%
INFO 05-17 07:18:35 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 18.9 tokens/s, Running: 45 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 2.8%, CPU KV cache usage: 0.0%
INFO:     ::1:33958 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 07:18:42 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 19.5 tokens/s, Running: 44 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 2.9%, CPU KV cache usage: 0.0%
INFO:     ::1:33900 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 07:18:49 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 19.0 tokens/s, Running: 43 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 3.0%, CPU KV cache usage: 0.0%
INFO:     ::1:33874 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 07:18:56 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 13.6 tokens/s, Running: 43 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 3.1%, CPU KV cache usage: 0.0%
INFO 05-17 07:19:01 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 24.9 tokens/s, Running: 43 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 3.1%, CPU KV cache usage: 0.0%
INFO:     ::1:33982 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 07:19:08 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 18.5 tokens/s, Running: 42 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 3.2%, CPU KV cache usage: 0.0%
INFO:     ::1:33858 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 07:19:15 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 18.1 tokens/s, Running: 40 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 3.3%, CPU KV cache usage: 0.0%
INFO:     ::1:33894 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 07:19:21 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 17.3 tokens/s, Running: 39 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 3.2%, CPU KV cache usage: 0.0%
INFO:     ::1:34014 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 07:19:28 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 17.0 tokens/s, Running: 39 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 3.3%, CPU KV cache usage: 0.0%
INFO 05-17 07:19:35 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 17.1 tokens/s, Running: 39 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 3.3%, CPU KV cache usage: 0.0%
INFO 05-17 07:19:42 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 17.1 tokens/s, Running: 39 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 3.5%, CPU KV cache usage: 0.0%
INFO 05-17 07:19:49 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 17.1 tokens/s, Running: 38 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 3.5%, CPU KV cache usage: 0.0%
INFO:     ::1:33844 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 07:19:56 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 11.2 tokens/s, Running: 38 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 3.7%, CPU KV cache usage: 0.0%
INFO:     ::1:33961 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:34018 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 07:20:02 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 22.1 tokens/s, Running: 35 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 3.5%, CPU KV cache usage: 0.0%
INFO:     ::1:33846 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 07:20:09 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 15.6 tokens/s, Running: 35 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 3.6%, CPU KV cache usage: 0.0%
INFO:     ::1:33975 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 07:20:16 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 10.8 tokens/s, Running: 34 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 3.5%, CPU KV cache usage: 0.0%
INFO 05-17 07:20:23 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 19.5 tokens/s, Running: 34 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 3.6%, CPU KV cache usage: 0.0%
INFO 05-17 07:20:29 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 15.1 tokens/s, Running: 34 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 3.8%, CPU KV cache usage: 0.0%
INFO 05-17 07:20:36 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 10.8 tokens/s, Running: 34 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 3.9%, CPU KV cache usage: 0.0%
INFO:     ::1:33868 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 07:20:43 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 18.9 tokens/s, Running: 33 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 3.8%, CPU KV cache usage: 0.0%
INFO 05-17 07:20:49 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 14.7 tokens/s, Running: 33 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 3.9%, CPU KV cache usage: 0.0%
INFO 05-17 07:20:56 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 10.8 tokens/s, Running: 33 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 3.9%, CPU KV cache usage: 0.0%
INFO 05-17 07:21:01 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 19.7 tokens/s, Running: 33 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 4.0%, CPU KV cache usage: 0.0%
INFO 05-17 07:21:07 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 14.8 tokens/s, Running: 32 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 4.1%, CPU KV cache usage: 0.0%
INFO:     ::1:33884 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 07:21:14 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 14.4 tokens/s, Running: 32 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 4.2%, CPU KV cache usage: 0.0%
INFO:     ::1:33988 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:33882 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 07:21:21 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 14.2 tokens/s, Running: 29 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 3.8%, CPU KV cache usage: 0.0%
INFO:     ::1:33936 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:33848 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 07:21:26 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 11.4 tokens/s, Running: 28 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 3.7%, CPU KV cache usage: 0.0%
INFO:     ::1:33872 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 07:21:31 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 15.0 tokens/s, Running: 27 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 3.7%, CPU KV cache usage: 0.0%
INFO 05-17 07:21:37 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 12.9 tokens/s, Running: 27 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 3.8%, CPU KV cache usage: 0.0%
INFO 05-17 07:21:44 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 12.9 tokens/s, Running: 27 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 3.9%, CPU KV cache usage: 0.0%
INFO:     ::1:33864 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 07:21:50 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 12.8 tokens/s, Running: 26 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 3.8%, CPU KV cache usage: 0.0%
INFO 05-17 07:21:56 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 9.1 tokens/s, Running: 26 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 3.8%, CPU KV cache usage: 0.0%
INFO 05-17 07:22:02 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 15.3 tokens/s, Running: 26 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 3.9%, CPU KV cache usage: 0.0%
INFO 05-17 07:22:09 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 12.4 tokens/s, Running: 26 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 4.0%, CPU KV cache usage: 0.0%
INFO:     ::1:33968 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 07:22:15 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 12.2 tokens/s, Running: 25 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 4.0%, CPU KV cache usage: 0.0%
INFO 05-17 07:22:21 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 12.0 tokens/s, Running: 25 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 4.0%, CPU KV cache usage: 0.0%
INFO 05-17 07:22:28 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 11.9 tokens/s, Running: 25 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 4.1%, CPU KV cache usage: 0.0%
INFO 05-17 07:22:34 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 11.9 tokens/s, Running: 25 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 4.1%, CPU KV cache usage: 0.0%
INFO:     ::1:33838 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 07:22:40 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 11.7 tokens/s, Running: 24 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 4.0%, CPU KV cache usage: 0.0%
INFO 05-17 07:22:46 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 8.9 tokens/s, Running: 24 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 4.2%, CPU KV cache usage: 0.0%
INFO 05-17 07:22:51 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 14.4 tokens/s, Running: 24 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 4.2%, CPU KV cache usage: 0.0%
INFO 05-17 07:22:56 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 9.6 tokens/s, Running: 24 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 4.3%, CPU KV cache usage: 0.0%
INFO 05-17 07:23:01 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 13.2 tokens/s, Running: 23 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 4.1%, CPU KV cache usage: 0.0%
INFO:     ::1:33918 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 07:23:07 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 11.0 tokens/s, Running: 23 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 4.2%, CPU KV cache usage: 0.0%
INFO:     ::1:33980 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 07:23:13 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 11.1 tokens/s, Running: 22 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 4.1%, CPU KV cache usage: 0.0%
INFO:     ::1:33902 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 07:23:19 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 10.8 tokens/s, Running: 21 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 4.0%, CPU KV cache usage: 0.0%
INFO 05-17 07:23:25 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 10.7 tokens/s, Running: 21 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 4.0%, CPU KV cache usage: 0.0%
INFO 05-17 07:23:31 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 10.7 tokens/s, Running: 21 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 4.1%, CPU KV cache usage: 0.0%
INFO 05-17 07:23:37 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 10.7 tokens/s, Running: 21 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 4.1%, CPU KV cache usage: 0.0%
INFO 05-17 07:23:43 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 10.6 tokens/s, Running: 20 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 4.0%, CPU KV cache usage: 0.0%
INFO:     ::1:33832 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 07:23:48 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 11.0 tokens/s, Running: 20 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 4.1%, CPU KV cache usage: 0.0%
INFO:     ::1:33942 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 07:23:54 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 10.8 tokens/s, Running: 19 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 3.9%, CPU KV cache usage: 0.0%
INFO 05-17 07:23:59 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 10.4 tokens/s, Running: 19 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 4.0%, CPU KV cache usage: 0.0%
INFO 05-17 07:24:05 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 10.4 tokens/s, Running: 18 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 3.8%, CPU KV cache usage: 0.0%
INFO:     ::1:34009 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:33898 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 07:24:11 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 10.7 tokens/s, Running: 17 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 3.7%, CPU KV cache usage: 0.0%
INFO 05-17 07:24:18 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 10.8 tokens/s, Running: 17 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 3.8%, CPU KV cache usage: 0.0%
INFO:     ::1:33984 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 07:24:24 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 10.9 tokens/s, Running: 16 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 3.6%, CPU KV cache usage: 0.0%
INFO 05-17 07:24:30 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 11.0 tokens/s, Running: 16 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 3.7%, CPU KV cache usage: 0.0%
INFO 05-17 07:24:35 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 11.1 tokens/s, Running: 16 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 3.7%, CPU KV cache usage: 0.0%
INFO 05-17 07:24:41 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 11.0 tokens/s, Running: 16 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 3.8%, CPU KV cache usage: 0.0%
INFO 05-17 07:24:47 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 11.2 tokens/s, Running: 16 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 3.9%, CPU KV cache usage: 0.0%
INFO:     ::1:33840 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 07:24:52 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 11.3 tokens/s, Running: 15 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 3.7%, CPU KV cache usage: 0.0%
INFO 05-17 07:24:57 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 11.4 tokens/s, Running: 15 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 3.7%, CPU KV cache usage: 0.0%
INFO 05-17 07:25:03 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 11.4 tokens/s, Running: 15 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 3.8%, CPU KV cache usage: 0.0%
INFO 05-17 07:25:08 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 11.5 tokens/s, Running: 15 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 3.9%, CPU KV cache usage: 0.0%
INFO 05-17 07:25:13 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 11.5 tokens/s, Running: 14 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 3.6%, CPU KV cache usage: 0.0%
INFO:     ::1:33991 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 07:25:18 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 10.7 tokens/s, Running: 14 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 3.6%, CPU KV cache usage: 0.0%
INFO:     ::1:33976 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 07:25:23 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 10.8 tokens/s, Running: 13 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 3.4%, CPU KV cache usage: 0.0%
INFO:     ::1:34012 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 07:25:29 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 10.7 tokens/s, Running: 12 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 3.2%, CPU KV cache usage: 0.0%
INFO 05-17 07:25:35 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 10.1 tokens/s, Running: 12 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 3.2%, CPU KV cache usage: 0.0%
INFO 05-17 07:25:41 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 10.1 tokens/s, Running: 12 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 3.3%, CPU KV cache usage: 0.0%
INFO:     ::1:33856 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 07:25:47 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 9.9 tokens/s, Running: 11 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 3.1%, CPU KV cache usage: 0.0%
INFO 05-17 07:25:53 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 9.3 tokens/s, Running: 11 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 3.2%, CPU KV cache usage: 0.0%
INFO:     ::1:33904 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 07:25:59 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 9.2 tokens/s, Running: 10 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 2.9%, CPU KV cache usage: 0.0%
INFO 05-17 07:26:05 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 8.5 tokens/s, Running: 10 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 3.0%, CPU KV cache usage: 0.0%
INFO 05-17 07:26:11 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 8.5 tokens/s, Running: 10 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 3.0%, CPU KV cache usage: 0.0%
INFO 05-17 07:26:17 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 8.5 tokens/s, Running: 10 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 3.1%, CPU KV cache usage: 0.0%
INFO 05-17 07:26:22 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 8.5 tokens/s, Running: 10 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 3.1%, CPU KV cache usage: 0.0%
INFO 05-17 07:26:28 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 8.6 tokens/s, Running: 10 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 3.2%, CPU KV cache usage: 0.0%
INFO:     ::1:33924 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:33880 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 07:26:34 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 7.9 tokens/s, Running: 8 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 2.6%, CPU KV cache usage: 0.0%
INFO 05-17 07:26:39 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 7.7 tokens/s, Running: 8 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 2.6%, CPU KV cache usage: 0.0%
INFO 05-17 07:26:44 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 7.7 tokens/s, Running: 8 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 2.7%, CPU KV cache usage: 0.0%
INFO 05-17 07:26:49 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 7.7 tokens/s, Running: 8 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 2.7%, CPU KV cache usage: 0.0%
INFO:     ::1:33842 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:34010 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 07:26:54 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 7.5 tokens/s, Running: 6 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 2.1%, CPU KV cache usage: 0.0%
INFO 05-17 07:27:00 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 6.6 tokens/s, Running: 6 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 2.1%, CPU KV cache usage: 0.0%
INFO:     ::1:34016 - "POST /v1/completions HTTP/1.1" 200 OK
INFO:     ::1:33966 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 07:27:05 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 6.5 tokens/s, Running: 4 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 1.5%, CPU KV cache usage: 0.0%
INFO 05-17 07:27:11 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 6.2 tokens/s, Running: 4 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 1.5%, CPU KV cache usage: 0.0%
INFO 05-17 07:27:16 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 5.6 tokens/s, Running: 4 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 1.5%, CPU KV cache usage: 0.0%
INFO 05-17 07:27:21 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 6.8 tokens/s, Running: 4 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 1.5%, CPU KV cache usage: 0.0%
INFO 05-17 07:27:26 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 6.2 tokens/s, Running: 4 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 1.6%, CPU KV cache usage: 0.0%
INFO 05-17 07:27:31 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 6.1 tokens/s, Running: 3 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 1.2%, CPU KV cache usage: 0.0%
INFO:     ::1:33965 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 07:27:37 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 5.8 tokens/s, Running: 3 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 1.2%, CPU KV cache usage: 0.0%
INFO 05-17 07:27:42 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 5.8 tokens/s, Running: 3 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 1.3%, CPU KV cache usage: 0.0%
INFO:     ::1:33946 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 07:27:47 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 5.8 tokens/s, Running: 2 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.9%, CPU KV cache usage: 0.0%
INFO 05-17 07:27:52 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 5.2 tokens/s, Running: 2 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.9%, CPU KV cache usage: 0.0%
INFO:     ::1:33910 - "POST /v1/completions HTTP/1.1" 200 OK
INFO 05-17 07:27:57 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 4.4 tokens/s, Running: 1 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.5%, CPU KV cache usage: 0.0%
INFO 05-17 07:28:02 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 4.0 tokens/s, Running: 1 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.5%, CPU KV cache usage: 0.0%
INFO 05-17 07:28:07 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 4.0 tokens/s, Running: 1 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.5%, CPU KV cache usage: 0.0%
INFO 05-17 07:28:12 metrics.py:276] Avg prompt throughput: 0.0 tokens/s, Avg generation throughput: 4.0 tokens/s, Running: 1 reqs, Swapped: 0 reqs, Pending: 0 reqs, GPU KV cache usage: 0.5%, CPU KV cache usage: 0.0%
INFO:     ::1:33878 - "POST /v1/completions HTTP/1.1" 200 OK
Results written to .artifact/benchmarks/results/03a50a0d-bc34-4766-adef-ee3e7e7625cf.jsonl
Killing process 90400...
/var/spool/slurmd/job3676347/slurm_script: line 46: 90366 Killed                  apptainer run --nv --home /mnt/petrelfs/huqinghao/xzyao/:/home/xiayao --bind /mnt/petrelfs/huqinghao/xzyao/code/vllm:/vllm --bind /mnt/petrelfs/huqinghao/xzyao/code/triteia:/triteia --env PYTHONPATH=/vllm:/triteia --env CUDA_LAUNCH_BLOCKING=1 --env TVM_TARGET=nvidia/nvidia-a100 --env TRANSFORMERS_OFFLINE=1 --env NUMEXPR_MAX_THREADS=128 --env USE_BITBLAS=1 --env BITBLAS_TARGET=nvidia/nvidia-a100 --env RAY_DEDUP_LOGS=0 --workdir /vllm /mnt/petrelfs/huqinghao/xzyao/images/vllm-0.4.1.sif python3 /vllm/vllm/entrypoints/openai/api_server.py --model /vllm/.idea/full_models/Llama-2-13b-hf --enable-delta --disable-log-requests --gpu-memory-utilization 0.85 --delta-modules delta-1=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-1 delta-2=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-2 delta-3=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-3 delta-4=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-4 delta-5=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-5 delta-6=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-6 delta-7=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-7 delta-8=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-8 delta-9=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-9 delta-10=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-10 delta-11=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-11 delta-12=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-12 delta-13=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-13 delta-14=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-14 delta-15=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-15 delta-16=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-16 delta-17=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-17 delta-18=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-18 delta-19=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-19 delta-20=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-20 delta-21=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-21 delta-22=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-22 delta-23=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-23 delta-24=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-24 delta-25=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-25 delta-26=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-26 delta-27=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-27 delta-28=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-28 delta-29=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-29 delta-30=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-30 delta-31=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-31 delta-32=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-32 delta-33=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-33 delta-34=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-34 delta-35=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-35 delta-36=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-36 delta-37=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-37 delta-38=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-38 delta-39=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-39 delta-40=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-40 delta-41=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-41 delta-42=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-42 delta-43=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-43 delta-44=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-44 delta-45=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-45 delta-46=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-46 delta-47=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-47 delta-48=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-48 delta-49=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-49 delta-50=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-50 delta-51=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-51 delta-52=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-52 delta-53=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-53 delta-54=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-54 delta-55=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-55 delta-56=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-56 delta-57=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-57 delta-58=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-58 delta-59=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-59 delta-60=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-60 delta-61=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-61 delta-62=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-62 delta-63=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-63 delta-64=/vllm/.idea/models/13b/lmsys.vicuna-13b-v1.5.4b75s128g-64 --max-deltas 24 --tensor-parallel-size 4 --enforce-eager --enable-prefetch
